js/main.js

```diff
@@ -118,6 +131,37 @@ function setStartup(lStartup){
     });
 }
 
+// Method to overwrite user config on mac installer
+function updateUserConfigMac() {
```

Comment:
Any reason we are creating a child process here in place of using node's fs api for manipulating files?